### ISSUE TITLE
Problem-ValueError: num_samples should be a positive integer value, but got num_samples=0

### ISSUE THREAD

**Issue by 15075171288** [qq456cvb/Point-Transformers#49]
I checked the path to my dataset，And there is nothing wrong，But I don't know why the dataset doesn't load.Can you give me some tips？

**Comment by 2108LEO** [comment]
yeah,i am meeting this problem too, how to do ? do you figure out ?
I need your help ,thanks.

**Comment by 2108LEO** [comment]
> I checked the path to my dataset，And there is nothing wrong，But I don't know why the dataset doesn't load.Can you give me some tips？
yeah,i am meeting this problem too, how to do ? do you figure out ?
I need your help ,thanks.

**Comment by 2108LEO** [comment]
> I checked the path to my dataset，And there is nothing wrong，But I don't know why the dataset doesn't load.Can you give me some tips？
ok，兄弟，我找到方法了，是数据集的问题，这个代码需要带normal后缀的数据集，我们一开始选错了数据集。

![Image](https://github.com/user-attachments/assets/1bed52d0-ca27-44df-806f-948558a16782)

**Comment by qq456cvb** [comment]
`num_samples=0` means the dataset loaded empty, so the path or folder layout doesn't match what the loader expects. Check that `root` points at the extracted `shapenetcore_partanno_segmentation_benchmark_v0_normal/` folder, and that it contains `synsetoffset2category.txt`, the per-category id folders, and `train_test_split/*.json`. If the zip extracted into an extra nested folder, point `root` one level deeper. A wrong split name will also give you 0 samples.
